### DISCUSSION TITLE
Bungeecord setting detected, but i didn't use bungee

### DISCUSSION THREAD

**Discussion by Scoooolzs** [games647/FastLogin#760] [Q&A] [answered]
Maybe the title says all.

![IMG_20220323_124255](https://user-images.githubusercontent.com/84376629/159631753-e91a86b2-4202-48f4-a874-73404d33442a.jpg)

**Comment by Scoooolzs** [comment, marked as answer]
Fixed with my hosting support.

**Reply by TuxCoding** [reply]
That's weird. I'm still interested in this. Does your server hoster use some custom reverse proxy?

**Reply by Scoooolzs** [reply]
> That's weird. I'm still interested in this. Does your server hoster use some custom reverse proxy?

No. But the hosting support says: "I accidentally turned on the bungeecord option", because the admin often visits my server.
